ros2_ws.Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM osrf/ros:humble-desktop
-LABEL Name=autonomous Version=0.0.1
-
-SHELL [ "/bin/bash", "-c" ]
-
-WORKDIR /ws
-COPY . .
-
-RUN source /opt/ros/humble/setup.bash && \
-    rosdep update && \
-    colcon build 
-
-ENTRYPOINT ["/ros_entrypoint.sh"]
-CMD [ "bash" ]
+FROM osrf/ros:humble-desktop
+LABEL Name=autonomous Version=0.0.1
+
+SHELL [ "/bin/bash", "-c" ]
+
+WORKDIR /ws
+COPY . .
+
+RUN source /opt/ros/humble/setup.bash && \
+    rosdep update && \
+    colcon build 
+
+RUN apt-get update && apt-get install -y \
+    python3-pip && \
+    apt-get clean

```

Comment:
Works perfictly but can you please just add this so that I don't have to source it every time I open a term

echo "source /opt/ros/humble/setup.bash" >> ~/.bashrc